eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -254,8 +256,15 @@ function DeployStressPackage(
             Write-Host "Building and pushing stress test docker image '$imageTag'"
             $dockerFile = Get-ChildItem $dockerFilePath
 
-            Run docker build -t $imageTag -f $dockerFile $dockerBuildFolder
+            $dockerBuildCmd = @("docker"; "build"; "-t"; $imageTag;"-f";$dockerFile)
+            foreach ($envVar in $dockerBuildConfig.scenario.GetEnumerator()) {
+                $dockerBuildCmd += "--build-arg"
+                $dockerBuildCmd += "$($envVar.Key)=$($envVar.Value)"
+            }
```

Comment:
They aren't really env variables so maybe a different variable name would be better?

Reply:
Also I think we should wrap the build arg value in single quotes in the built string, just in case people have some values like semicolons, &&, etc. (which I imagine they will if they are passing shell command args down).